.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -4,24 +4,52 @@ about: Request our thoughts on your suggestion for a new feature for Homebrew.
 
 ---
 
-# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
+<details>
+<summary><strong>Guidelines</strong></summary>
+Thank you for making a feature suggestion! :heart:
 
-<!-- Please fill these sections with the relevant information: -->
+**Please fill out all the sections below with the relevant information.**
 
-# A detailed description of the proposed feature
+We will review your suggestion and let you know our decision as soon as we can.
+Note, that we will close this issue if neither you nor we have the time to work on it.
 
-<!-- replace me -->
+Homebrew is a non-profit project run entirely by unpaid volunteers.
+We, therefore, reserve the right to close your issue without comment if you
+delete, do not read, or do not fill out the issue checklist below and provide
+_**all** information requested_. Note, that if you repeatedly fail to fill out
+the issue template, we will be forced to block you from ever submitting issues
+to Homebrew.
 
-# The motivation for the feature
+Thank you for your understanding,<br>
+\- Homebrew maintainers
+</details>
 
-<!-- replace me -->
 
-# How the feature would be relevant to at least 90% of Homebrew users
+## Description
+_A detailed description of the proposed feature._
 
-<!-- (if it's not: do not open a feature request) -->
+replace me
 
-# What alternatives to the feature have been considered
 
-<!-- replace me -->
+## Motivation
+_The motivation for the feature._
 
-<!-- We will close this issue or ask you to create a pull-request if it's something the maintainers are not actively planning to work on. -->
+replace me
+
+
+## Relevance
+_How the proposed feature would be relevant to at least 90% of Homebrew users._
+
+replace me
+
+
+## Alternatives
+_What alternatives to the feature have been considered._
+
+replace me
+
+
+## Your Commitment
+_Please indicate if you're willing to implement this feature._
```

Comment:
This is somewhat irrelevant because even if they say they will: that doesn't mean we're any more or less likely to close or accept it or that their implementation will be acceptable.

Reply:
This is not related to the decision-making step but to the one after it if the suggestion is accepted. We can:
- add "if your suggestion is accepted"
- remove this part